api/v1alpha1/module_types.go

```diff
@@ -125,6 +125,11 @@ type ModuleSpec struct {
 	// More info: https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/
 	// +optional
 	ServiceAccountName string `json:"serviceAccountName,omitempty"`
+
+	// +optional
+	// ServiceAccountName is the name of the ServiceAccount to use to run this pod.
```

Comment:
The comment does not match the field's name.

Reply:
Fixed.